lib/graphql_rails/model/configuration.rb

```diff
@@ -42,7 +42,12 @@ def graphql_type
       def connection_type
         @connection_type ||= begin
           graphql_type.define_connection do
-            field :total, types.Int, resolve: ->(obj, _args, _ctx) { obj.nodes.size }
+            field(:total, types.Int, resolve: lambda do |obj, _args, _ctx|
```

Comment:
those parentheses looks weird. You could still keep field definition in one line by moving `lambda` in to constant, like:
```
class Configuration
  COUNT_TOTAL_ENTRIES = lambda do |obj, _args, _ctx|
    obj_nodes = obj.nodes
    obj_nodes = obj_nodes.except(:offset) if obj_nodes.is_a?(ActiveRecord::Relation)
    obj_nodes.size
  end
  
  def connection_type
    @connection_type ||= begin
      graphql_type.define_connection do
         field(:total, types.Int, resolve: COUNT_TOTAL_ITEMS)
      end
  end
end
```

Reply:
Yes, it looks weird. Thanks for the recommendation.